.github/workflows/black.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6]
+        python-version: [3.9]
```

Comment:
Once we are on it, and taking into account that this one only affects formatting, why don't we upgrade to latest Python 3.11?